securitycenter/snippets_v2/snippets_mute_config_v2.py

```diff
@@ -0,0 +1,183 @@
+        '-resource.type:"compute"'
+    )
+    mute_config.type = "STATIC"
+
+    request = securitycenter_v2.CreateMuteConfigRequest()
+    request.parent = parent_path+"/locations/"+location_id
+    request.mute_config_id = mute_config_id
+    request.mute_config = mute_config
+
+
+    mute_config = client.create_mute_config(request=request)
+    print(f"Mute rule created successfully: {mute_config.name}")
+    return mute_config
+
+# [END securitycenter_create_mute_config_v2]
+
+
+# [START securitycenter_delete_mute_config_v2]
+def delete_mute_rule(parent_path: str, location_id: str, mute_config_name: str) -> None:
+    """
+    Deletes a mute configuration given its resource name.
+    Note: Previously muted findings are not affected when a mute config is deleted.
+    Args:
+         parent_path: use any one of the following options:
+                     - organizations/{organization_id}
+                     - folders/{folder_id}
+                     - projects/{project_id}
+        mute_config_name: Set a unique id; max of 63 chars.
+    """
+    from google.cloud import securitycenter_v2
+
+    client = securitycenter_v2.SecurityCenterClient()
+
+    request = securitycenter_v2.DeleteMuteConfigRequest()
+    request.name = parent_path+"/locations/"+location_id+"/muteConfigs/"+mute_config_name
+
+    client.delete_mute_config(request)
+    print(f"Mute rule deleted successfully: {mute_config_name}")
+
+
+# [END securitycenter_delete_mute_config_v2]
+
+
+# [START securitycenter_get_mute_config_v2]
+def get_mute_rule(parent_path: str, location_id: str, mute_config_name: str) -> None:
+    """
+    Retrieves a mute configuration given its resource name.
+    Args:
+        parent_path: use any one of the following options:
+                     - organizations/{organization_id}
+                     - folders/{folder_id}
+                     - projects/{project_id}
+        mute_config_name: Set a unique id; max of 63 chars.
+    """
+    from google.cloud import securitycenter_v2
+
+    client = securitycenter_v2.SecurityCenterClient()
+
+    request = securitycenter_v2.GetMuteConfigRequest()
+    request.name = parent_path+"/locations/"+location_id+"/muteConfigs/"+mute_config_name
+
+    mute_config = client.get_mute_config(request)
+    print(f"Retrieved the mute rule: {mute_config.name}")
+    return mute_config
+
+# [END securitycenter_get_mute_config_v2]
+
+
+# [START securitycenter_list_mute_configs_v2]
+def list_mute_rules(parent: str, location_id: str) -> None:
+    """
+    Listing mute configs at organization level will return all the configs
+    at the org, folder and project levels.
+    Similarly, listing configs at folder level will list all the configs
+    at the folder and project levels.
+    Args:
+        parent: Use any one of the following resource paths to list mute configurations:
+                - organizations/{organization_id}
+                - folders/{folder_id}
+                - projects/{project_id}
```

Comment:
ditto: `location_id` is missing from docstring

Reply:
addressed